src/Model/GServer.php

```diff
@@ -576,10 +598,15 @@ public static function detect(string $url, string $network = '', bool $only_node
 			}
 		}
 
-		if (!empty($serverdata['network']) && in_array($serverdata['network'], [Protocol::DFRN, Protocol::DIASPORA])) {
```

Comment:
I think I have added this to prevent a `E_NOTICE` of a missing array element or so.

Reply:
this check is not needed anymore.